### PR TITLE
Added support for spawners, solves problem with WildStacker

### PR DESCRIPTION
**Pull request by Paul19988**
- Solves issue where players can take spawners that have been stacked together from areas they don't have access to.
- Changed pom.xml to compile plugin with version in .jar file name

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -53,7 +53,7 @@
     </repositories>
 
     <build>
-        <finalName>${project.name}</finalName>
+        <finalName>${project.name}-${project.version}</finalName>
```
Comment (RoboMWM) [inline pom.xml:56]: no

**Review by RoboMWM — CHANGES_REQUESTED** [review]
how does this "add support for spawners" and ~~"solves problem with wildstacker"??~~

Ok, so I have no idea what wildstacker is... so now people stack spawners or something?

I'd rather not end up right click checking every block just because some rando plugins decide to care about blocks that vanilla doesn't make interactable in the first place. I think it should go the other way around IMO, or do something like fire a fake blockbreakevent to easily check for permission.

**Comment by Paul19988** [comment]
Plugins like WildStacker add the ability to stack spawners which gives them an interactable inventory, which in turn allows players to bypass and steal within a claim.

Edit:
I'll go ahead and drop them a line to see if they'll make a change similar to how you have described in your edited message above.

**Comment by RoboMWM** [comment]
Was editing my comment while you posted that.

Ok yea it's on the plugin if they want to deal with making blocks interactable. Many ways you can go about it, including adding your own ownership system (like many shop plugins do). See another way proposed in my edited comment.

**Pull request closed without merging**